.circleci/config.yml

```diff
@@ -259,4 +269,4 @@ workflows:
             - build-deploy-mosaic-production
           filters:
             branches:
-              only: main
+              only: main
```

Comment:
Doesn't matter too much but missing new line at the end of this file.

Reply:
Good spot, have added this in.